### DISCUSSION TITLE
Library update / only changed files not working

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#967] [Fixed Issues]
Running version 202207081

After selecting library update / changed files only, the system looks like it is performing an update as it normally would.  

Shared data is off.

After about 10 seconds, the system then displays the "update complete" message and then posts the same message again 5 seconds later.   

After reviewing the library, it is clear that nothing has been updated (all counts remain unchanged, despite adding new music files to the library before issuing a library update request).

Library update / rebuilt entire database function is working fine (my current workaround). 

I noticed this behavior a few updates ago, but cannot provide a specific version when the problem first started occurring.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Please try today's update.

**Reply by Sagnasty** [reply]
Fixed - Thanks Rern.
